Assets/_Application/Layers/LayerTypes/HierarchicalObject/HierarchicalObjectLayerGameObject.cs

```diff
@@ -58,6 +61,7 @@ private BoundingBox CalculateWorldBoundsFromRenderers()
 
         protected virtual void Awake()
         {
+            snappingCullingMask = (1 << LayerMask.NameToLayer("Terrain")) | (1 << LayerMask.NameToLayer("Buildings"));
```

Comment:
Dit werkt alleen met onze eigen gemaakte interne lagen - als je een terreinlaag of gebouwen importeert vanuit 3D Tiles (zoals Tygron gebouwen) dan werkt dit niet. Maar dit hangt vast samen met de story om hoogte maaiveld te bepalen?